### PR TITLE
N31: Name Needed #949

### PR DESCRIPTION
**Pull request by DomWilliams0**
Adds progress on Name Needed. Thank you!

#949

### CODE REVIEW

File: content/news/031/index.md [hunk 1]
```diff
@@ -68,6 +68,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Game Updates
 
+### [Name Needed][name-needed]
+
+![The player orders a selection of blocks to be broken, and a brick wall to be
+built.](name-needed.gif)
+
+_The player orders a selection of blocks to be broken, and a brick wall to be
+built._
```
Comment (17cupsofcoffee) [inline content/news/031/index.md:77]: Need to remove this line to get the CSS to do caption formatting:
```suggestion
![The player orders a selection of blocks to be broken, and a brick wall to be
built.](name-needed.gif)
_The player orders a selection of blocks to be broken, and a brick wall to be
built._
```

File: content/news/031/index.md [hunk 2]
```diff
@@ -68,6 +68,32 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+
+[Name Needed][name-needed] by [@DomWilliams0][domwilliams-github] is a one man
```
Comment (17cupsofcoffee) [inline content/news/031/index.md:79]:
```suggestion
[Name Needed][name-needed] by [@DomWilliams0][domwilliams-github] is a one-man
```

**Review by 17cupsofcoffee — APPROVED** [review]
Content looks good.

The GIF is a little on the big side (though still just within our usual limit), but EZGif doesn't seem to be able to improve it. If you can optimize it on your end, it'd be appreciated, but not essential!

**Comment by DomWilliams0** [comment]
I shrunk it down to 2.1MB by resizing it a bit, hope that's more manageable

**Review by 17cupsofcoffee — APPROVED** [review]

**Review by AngelOnFira — APPROVED** [review]

**Pull request merged**